### PR TITLE
fix(Switch): adjust design

### LINKED ISSUE #2664: Switch adjustments

### PR DESCRIPTION
**Pull request by eirikbacker**
- Fixes #2664

### CODE REVIEW

File: packages/css/src/input.css [hunk 1]
```diff
@@ -231,28 +224,19 @@
...
     &:checked {
-      --dsc-input-color: var(--dsc-input-border-color);
       --circle-position: right;
     }
 
-    &[readonly] {
-      --circle-color: var(--dsc-input-color--readonly);
-    }
-
+    /* TOBIAS: Do we need this after adjusting colors (as we allready have this defined in radio/check)? */
```
[the remark this answers is not in the document]
Comment (Barsnes) [reply packages/css/src/input.css]: This was adjusted by me in this branch

**Comment by Barsnes** [comment]
It looks like this is missing the lock icon when in read-only mode, but this is also the case for `Checkbox` and `Radio`, so I suggest we solve all of those in a separate PR after this
image

**Comment by eirikbacker** [comment]
> It looks like this is missing the lock icon when in read-only mode, but this is also the case for `Checkbox` and `Radio`, so I suggest we solve all of those in a separate PR after this image

Yeah, talked to @Febakke about this, as radio/checkbox does not have lock right now, we await doing it for Switch too. He has texted Altinn Studio (who use a lot of readonly and switch) and asked about their opinion. Supporting doing this in another PR if it turns out to be something we want :)

**Comment by Barsnes** [comment]
> > It looks like this is missing the lock icon when in read-only mode, but this is also the case for `Checkbox` and `Radio`, so I suggest we solve all of those in a separate PR after this.
> 
> Yeah, talked to @Febakke about this, as radio/checkbox does not have lock right now, we await doing it for Switch too. He has texted Altinn Studio (who use a lot of readonly and switch) and asked about their opinion. Supporting doing this in another PR if it turns out to be something we want :)

Great. If this needs investegating, do we have an issue, or should one be made?

**Review by Barsnes — APPROVED** [review]

**Pull request merged**